content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml

```diff
@@ -114,6 +114,48 @@
                             </column>
                         </items>
                     </socialmedia>
+                    <singlepageapps
```

Comment:
Shouldn't this section be relocated under `_cq_design_dialog`?

Reply:
yes. It is just easier to test it this way and see how the field types are behaving. It is also to gather the feedback about the names / structure.

Sorry I didn't put the comment about that upfront!